src/LuaTransformer.ts

```diff
@@ -3320,34 +3316,99 @@ export class LuaTransformer {
     }
 
     // Transpile a Math._ property
-    public transformMathExpression(identifier: ts.Identifier): tstl.TableIndexExpression {
-        const translation = {
-            PI: "pi",
-            abs: "abs",
-            acos: "acos",
-            asin: "asin",
-            atan: "atan",
-            ceil: "ceil",
-            cos: "cos",
-            exp: "exp",
-            floor: "floor",
-            log: "log",
-            max: "max",
-            min: "min",
-            pow: "pow",
-            random: "random",
-            round: "round",
-            sin: "sin",
-            sqrt: "sqrt",
-            tan: "tan",
-        };
+    public transformMathExpression(identifier: ts.Identifier): tstl.Expression {
+        const name = identifier.escapedText as string;
+        switch (name) {
+            case "PI":
+                const property = tstl.createStringLiteral("pi");
+                const math = tstl.createIdentifier("math");
+                return tstl.createTableIndexExpression(math, property, identifier);
```

Comment:
Why are you treating pi different from E?

Reply:
Since lua has it's own constant for pi, it's probably best to use that.

Reply:
Ah, if that's the reasoning I'm fine with that 👍